.github/workflows/pre-merge-ci.yml

```diff
@@ -3,6 +3,13 @@ name: Pre-Merge Integration tests
 on: 
   pull_request:
     types: [ labeled, opened, synchronize, reopened ]
+    paths:
+      - 'src/golang/**'
+      - 'src/python/**'
+      - 'integration_tests/**'
+      - 'sdk/aqueduct/**'
+      - '.github/workflows/integration-tests.yml'
+      - '.github/workflows/pre-merge-ci.yml'
```

Comment:
is this ok? Or do the repo settings force this exact action to pass before anything can be merged.

Reply:
Yea, the repo settings are strict for all PRs regardless of what code is being touched. We should remove these paths.